ldms/src/store/store_flatfile/store_flatfile.c

```diff
@@ -91,10 +92,9 @@ struct flatfile_metric_store {
 };
 
 struct flatfile_store_instance {
-	struct ldmsd_store *store;
+	struct ldmsd_cfgobj_store *store_cfg;
```

Comment:
ldmsd_cfgobj_store should not be exposed to the plugins. Fortunately, this is not really used in this code, and can be easily removed.